modules/custom/d_product/d_product_init/pages/product_2.yml

```diff
@@ -1,16 +1,14 @@
 info:
   type: d_product
-  title: Classic Black Piano
+  title: Super Charged Engine
 field_product_image:
   type: image
   data:
-    - file: d_product_init/demo_business_1.jpg
+    - file: d_product_init/d_product_init_super-charged-engine-1.jpg
```

Comment:
Usunięty fizycznie obraz `demo_business_1.jpg` jest używany w 6 innych YMLach. To może dotyczyć również pozostałych usuniętych plików.